src/Dev/SapphireTest.php

```diff
@@ -1232,9 +1231,9 @@ public function logInWithPermission($permCode = "ADMIN")
                 $group->Permissions()->add($permission);
             }
 
-            $member = DataObject::get_one('SilverStripe\\Security\\Member', array(
-                '"Member"."Email"' => "$permCode@example.org"
-            ));
+            $member = Member::get()->filter([
```

Comment:
We made a mistake in 3.0 or 3.1 being gung ho about deprecated get_one without a replacement for its caching behaviour.